app/api/activitylog/specs/activitylog.spec.js

```diff
@@ -31,7 +31,7 @@ describe('activitylog', () => {
       expect(entries.length).toBe(5);
```

Comment:
From the description of the text ` should return the entries, excluding certain routes`. I think we have just one special case `{ $ne: '/api/semantic-search/notify-updates' }`. IMO we should not log these entries nor filter them.

Reply:
Sure, I didn't realize that. This test is not needed anymore given we are not logging this special case and we'll delete the old entries.